launcher/container_runner.go

```diff
@@ -159,6 +160,20 @@ func NewRunner(ctx context.Context, cdClient *containerd.Client, token oauth2.To
 		specOpts = append(specOpts, oci.WithDevShmSize(launchSpec.DevShmSize))
 	}
 
+	if launchSpec.Experiments.EnableGpuDriverInstallation && launchSpec.InstallGpuDriver {
+		appendGpuDriverMounts(mounts)
+		specOpts = append(specOpts, oci.WithMounts(mounts))
+
+		gpuDeviceFiles, err := listFilesWithPrefix("/dev", "nvidia")
```

Comment:
consider using https://pkg.go.dev/path/filepath#WalkDir to get all nvidia devices, may make the code more concise